src/components/ui/Switch/switch.scss

```
@@ -0,0 +1,38 @@
+.Layer__Switch {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  font-size: var(--text-md);
```

No need to enforce a font size, that can be done by the child.
```suggestion
  align-items: center;
```